vendor/engines/bulk_email/app/assets/stylesheets/bulk_email/application.scss

```diff
@@ -6,6 +6,11 @@
 
 }
 
+.bulk_email_recipient_list {
+  max-height: 100px;
```

Comment:
I guess it is magic. I want to limit the height of this list and 100 looked reasonable. I'm open to alternatives.